src/inmanta/app.py

```diff
@@ -218,6 +218,18 @@ def compiler_config(parser: ArgumentParser) -> None:
     parser.add_argument("--password", dest="password", help="The password of the server")
     parser.add_argument("--ssl", help="Enable SSL", action="store_true", default=False)
     parser.add_argument("--ssl-ca-cert", dest="ca_cert", help="Certificate authority for SSL")
+    parser.add_argument(
+        "--export-compile-data",
```

Comment:
shouldn't it be beter to name it `--json` like for example the same option on the 'ip' command?

Reply:
I have no preference at all here, so I'll apply the suggestion.